assets/chat/js/window.js

```diff
@@ -128,12 +128,19 @@ class ChatWindow extends EventEmitter {
       }
 
       if (message.user?.isSystem()) {
-        return;
+        continue;
```

Comment:
Nope, completely unrelated. It's just a small bugfix: any message that comes after a system broadcast ("Destiny is live AngelThump", "Now hosting forsen", stuff like that) won't get processed by the function at all with a `return` there, since that means the function quits earlier than expected.